src/components/Notification/index.js

```diff
@@ -0,0 +1,174 @@
+import './notification.scss'
+import { captalize } from '../../utils/captalize'
+import SbIcon from '../Icon'
+import SbBadge from '../Badge'
+
+const SbNotification = {
+  name: 'SbNotification',
+
+  props: {
+    status: {
+      type: String,
+      default: 'general'
+    },
+    title: {
+      type: String,
+      default: 'notification'
+    },
+    description: {
+      type: String,
+      default: null
+    },
+    link: {
+      type: String,
+      default: null
+    },
+    linkName: {
+      type: String,
+      default: null
+    },
+    isExpandable: {
+      type: Boolean,
+      default: false
+    },
+    isFull: {
+      type: Boolean,
+      default: false
+    }
+  },
+
+  data () {
+    return {
+      expandle: false
+    }
+  },
+
+  methods: {
+    expandleNotification () {
```

Comment:
Suggestion: you can refactor this method to simple toggle boolean state, like this:

```js
expandleNotification () {
  this.expandle = !this.expandle
}
```

Reply:
ok